package.json

```diff
@@ -87,6 +89,16 @@
 				"command": "gitpod.exportLogs",
 				"category": "Gitpod",
 				"title": "Export all logs"
+			},
+			{
+				"command": "gitpod.releaseNote",
+				"category": "Gitpod",
+				"title": "Show Release Note"
+			},
+			{
+				"command": "gitpod.cleanReleaseNoteCache",
+				"category": "Gitpod",
```

Comment:
Debug actions should go under `Developer`

Reply:
I will remove it before merge